packages/block-library/src/site-logo/index.php

```diff
@@ -23,7 +23,22 @@ function render_block_core_site_logo( $attributes ) {
 
 	add_filter( 'wp_get_attachment_image_src', $adjust_width_height_filter );
 	$custom_logo = get_custom_logo();
-	$classnames  = array();
+
+	if ( ! $attributes['isLink'] ) {
+		/* Remove the link */
```

Comment:
Should probably use `//` style for comments...
```suggestion
		// Remove the link.
```